### PR TITLE
[ES] Update `Play style`

### PR DESCRIPTION
**Pull request by Ladwick**
Article update –  `Estilo de juego`

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Play_style/es.md [hunk 1]
```diff
@@ -1,141 +1,173 @@
...
-
-En esta página puedes conocer las diferentes formas en las que se puede jugar en cada modo de juego.
+En esta página se detallan las distintas formas que los jugadores pueden elegir para jugar en sus respectivos modos de juego.
 
 ## ![](/wiki/shared/mode/osu.png) osu!
 
 ### Solo con el ratón
 
-Tiempo atrás, únicamente existía la opción de jugar solo con el ratón, hasta que se incluyeron nuevas formas de jugar.
+También conocido como *solo ratón*, los jugadores que emplean este método desplazan el cursor con un ratón mientras pulsan las notas, siguen los sliders y giran los spinners usando solamente los botones izquierdo y derecho del ratón.
```
Comment (agatemosu) [inline wiki/Gameplay/Play_style/es.md:9]:
```suggestion
También conocido como *ratón puro*, los jugadores que emplean este método desplazan el cursor con un ratón mientras pulsan las notas, siguen los sliders y giran los spinners usando solamente los botones izquierdo y derecho del ratón.
```

File: wiki/Gameplay/Play_style/es.md [hunk 2]
```diff
@@ -1,141 +1,173 @@
...
-En pantalla, los clics con el botón izquierdo y derecho se muestran como `M1` y `M2` respectivamente.
+*Nota: En pantalla, los clics con el botón izquierdo y derecho se muestran como `M1` y `M2` respectivamente.*
+
+### Ratón doble
 
-### Con dos ratones
+El ratón doble es un estilo de juego en el cual un jugador tiene un ratón en cada mano, usando uno para pulsar y otro para mover el cursor por la pantalla.
 
-Un jugador puede utilizar un segundo ratón, de modo que use uno de ellos para hacer clic y otro para mover el cursor por la pantalla.
+Era considerado un estilo de juego *ridículo* por la mayoría de jugadores hasta que en mayo de 2013, el jugador canadiense ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) empezó a obtener grandes puntuaciones como [Hatsune Miku - With a Dance Number (val0108)](https://osu.ppy.sh/beatmapsets/29691) and [The Quick Brown Fox - The Big Black (Blue Dragon)](https://osu.ppy.sh/beatmapsets/41823) mientras jugaba con dos ratones.
```
Comment (agatemosu) [inline wiki/Gameplay/Play_style/es.md:21]:
```suggestion
Era considerado un estilo de juego *ridículo* por la mayoría de jugadores hasta que en mayo de 2013, el jugador canadiense ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) empezó a obtener grandes puntuaciones como [Hatsune Miku - With a Dance Number (val0108)](https://osu.ppy.sh/beatmapsets/29691) y [The Quick Brown Fox - The Big Black (Blue Dragon)](https://osu.ppy.sh/beatmapsets/41823) mientras jugaba con dos ratones.
```

File: wiki/Gameplay/Play_style/es.md [hunk 3]
```diff
@@ -1,141 +1,173 @@
...
+
+Los controladores de arcade de otros juegos rítmicos tales como DJMAX, O2Jam, Beatmania IIDX, etc. son un estilo de juego hecho para veteranos experimentados en juegos rítmicos de tipo transportador.
+
+*Nota: Ya que el uso de 8 teclas requiere la decisión del mapper o el uso de modificadores del juego, la mayoría de jugadores que tienen estos controles de arcade pueden manejar hasta 7 teclas (máximo por defecto) y 1 especial.*
 
-El teclado fue una vez la única manera de jugar a osu!mania hasta que otros métodos fueron incluidos.
+### Dance pad
 
-Los jugadores asignan las teclas que prefieran en el menú de opciones, que varían en función del número de teclas.
+Los dance pads son controladores para juegos como la serie [Dance Dance Revolution](https://wikipedia.org/wiki/Dance_Dance_Revolution) (DDR), y la serie [Pump It Up](https://en.wikipedia.org/wiki/Pump_It_Up_(video_game_series)) (PIU). ([StepMania](https://wikipedia.org/wiki/StepMania) es un clon semi-conocido de estos dos juegos, mantenido por la comunidad).
```
Comment (agatemosu) [inline wiki/Gameplay/Play_style/es.md:162]:
```suggestion
Los dance pads son controladores para juegos como la serie [Dance Dance Revolution](https://es.wikipedia.org/wiki/Dance_Dance_Revolution_(serie)) (DDR), y la serie [Pump It Up](https://es.wikipedia.org/wiki/Pump_It_Up_(serie_de_videojuegos)) (PIU). ([StepMania](https://es.wikipedia.org/wiki/StepMania) es un clon semi-conocido de estos dos juegos, mantenido por la comunidad).
```

File: wiki/Gameplay/Play_style/es.md [hunk 4]
```diff
@@ -1,141 +1,173 @@
...
+
+*Nota: Para la asignación de teclas, el valor por defecto es `Z`, `X`, `C`, y `V` (KDDK, o BRRB).*
 
-El jugador utiliza las teclas asignadas para tocar sobre el tambor, por defecto: `X` y `C` para tocar las notas rojas del centro del tambor, también conocidas como "Don"; y `Z` y `V` para tocar las notas azules del borde del tambor, también conocidas como "Katsu" o "Kat".
+### Control de taiko
 
-Hay dos estilos principales de juego en función de la asignación de teclas: KDDK (Azul-Rojo-Rojo-Azul) que es el estilo por defecto y DDKK/KKDD (RRAA/AARR).
-Hay otras combinaciones, pero estas dos son las principales y cada una tiene sus virtudes y sus defectos.
+El control de taiko es el estilo de juego por defecto y el control en la serie de juegos [Taiko no Tatsujin](https://wikipedia.org/wiki/Taiko_no_Tatsujin) ([Taiko Drum Master](https://wikipedia.org/wiki/Taiko:_Drum_Master) en los Estados Unidos).
```
Comment (agatemosu) [inline wiki/Gameplay/Play_style/es.md:98]:
```suggestion
El control de taiko es el estilo de juego por defecto y el control en la serie de juegos [Taiko no Tatsujin](https://es.wikipedia.org/wiki/Taiko_no_Tatsujin) ([Taiko Drum Master](https://en.wikipedia.org/wiki/Taiko:_Drum_Master) en los Estados Unidos).
```

File: wiki/Gameplay/Play_style/es.md [hunk 5]
```diff
@@ -1,141 +1,173 @@
...
+## Otros estilos
+
+- **Solo ratón**: Este es un estilo de nicho en el cual los jugadores controlan todos los movimientos y el dash del catcher únicamente a través de las entradas del ratón. Aunque esta opción solo está disponible en ratones con botones `M4` y `M5` (los cuales usualmente se encuentra en la parte lateral izquierda del ratón).
 
 ## ![](/wiki/shared/mode/mania.png) osu!mania
 
 ### Teclado
 
-[Guía externa para teclados mecánicos (en inglés)](http://www.daskeyboard.com/blog/mechanical-keyboard-guide/)
+*Nota: Para más información acerca de teclados mecánicos, véase [guía externa para teclados mecánicos (en inglés)](http://www.daskeyboard.com/blog/mechanical-keyboard-guide/).*
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:148]:
```suggestion
*Nota: Para más información acerca de los teclados mecánicos, véase la [guía externa para teclados mecánicos (en inglés)](http://www.daskeyboard.com/blog/mechanical-keyboard-guide/).*
```

File: wiki/Gameplay/Play_style/es.md [hunk 6]
```diff
@@ -1,141 +1,173 @@
...
+
+El teclado fue una vez la *única* manera de jugar osu!mania hasta que se les dio soporte a otros métodos con el tiempo.
+
+Para jugar con teclado, los jugadores deben presionar las teclas correspondientes a su ubicación en el teclado del juego. Por defecto estas teclas son `D`, `F`, `J`, y `K`. La ventaja de este estilo de juego es que se puede adaptar a básicamente cualquier configuración de teclas.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:152]:
```suggestion
Para jugar con teclado, los jugadores deben presionar las teclas correspondientes en el teclado del juego. De forma predeterminada, estas teclas son `D`, `F`, `J`, y `K`. La ventaja de este estilo de juego es que se puede adaptar a básicamente cualquier configuración de teclas.
```

File: wiki/Gameplay/Play_style/es.md [hunk 7]
```diff
@@ -1,141 +1,173 @@
...
+
+### Controlador de arcade
+
+Los controladores de arcade de otros juegos rítmicos tales como DJMAX, O2Jam, Beatmania IIDX, etc. son un estilo de juego hecho para veteranos experimentados en juegos rítmicos de tipo transportador.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:156]:
```suggestion
Los controladores de arcade de otros juegos de ritmo como DJMAX, O2Jam, Beatmania IIDX, etc., son un estilo de juego hecho para veteranos experimentados en juegos de ritmo de tipo transportador.
```

File: wiki/Gameplay/Play_style/es.md [hunk 8]
```diff
@@ -1,141 +1,173 @@
...
+
+*Nota: Ya que el uso de 8 teclas requiere la decisión del mapper o el uso de modificadores del juego, la mayoría de jugadores que tienen estos controles de arcade pueden manejar hasta 7 teclas (máximo por defecto) y 1 especial.*
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:158]:
```suggestion
*Nota: Ya que el uso de 8 teclas requiere la decisión del mapper o el uso de modificadores del juego, la mayoría de jugadores que tienen estos controles de arcade pueden manejar hasta 7 teclas (como máximo de forma predeterminada) y 1 especial.*
```

File: wiki/Gameplay/Play_style/es.md [hunk 9]
```diff
@@ -1,141 +1,173 @@
...
 
-El teclado fue una vez la única manera de jugar a osu!mania hasta que otros métodos fueron incluidos.
+### Dance pad
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:160]:
```suggestion
### Pad de baile
```

File: wiki/Gameplay/Play_style/es.md [hunk 10]
```diff
@@ -1,141 +1,173 @@
...
 
-Los jugadores asignan las teclas que prefieran en el menú de opciones, que varían en función del número de teclas.
+Los dance pads son controladores para juegos como la serie [Dance Dance Revolution](https://es.wikipedia.org/wiki/Dance_Dance_Revolution_(serie)) (DDR), y la serie [Pump It Up](https://es.wikipedia.org/wiki/Pump_It_Up_(serie_de_videojuegos)) (PIU). ([StepMania](https://es.wikipedia.org/wiki/StepMania) es un clon semi-conocido de estos dos juegos, mantenido por la comunidad).
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:162]:
```suggestion
Los pad de baile son controles para juegos como la serie [Dance Dance Revolution](https://es.wikipedia.org/wiki/Dance_Dance_Revolution_(serie)) (DDR) y la serie [Pump It Up](https://es.wikipedia.org/wiki/Pump_It_Up_(serie_de_videojuegos)) (PIU). ([StepMania](https://es.wikipedia.org/wiki/StepMania) es un clon semiconocido mantenido por la comunidad de estos dos juegos).
```

File: wiki/Gameplay/Play_style/es.md [hunk 11]
```diff
@@ -1,141 +1,173 @@
...
 
-Puede ser utilizado para cualquier número de teclas, así que puede resultar muy cómodo usar un teclado, aunque tiene sus desventajas.
+Para jugar con un dance pad, los jugadores «bailan» en el dance pad para introducir comandos. Los pads DDR tienen 4 paneles: Arriba, Abajo, Izquierda y Derecha. Los pads DDR Solo añaden paneles Superior Izquierdo y Superior Derecho. Los pads PIU tienen 5 paneles: Superior Izquierdo, Superior Derecho, Inferior Izquierdo, Inferior Derecho y Centro. También existen pads de 9 paneles con el fin de ser compatibles tanto con PIU como con DDR sin tener que cambiar el pad. Los paneles DDR y PIU normales son usados a menudo en modo doble en sus respectivos juegos, donde un jugador usa 2 pads para una canción. Los juegos DDR Solo eran únicamente de un jugador, por lo que no son dobles comúnmente.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:164]:
```suggestion
Para jugar con un pad de baile, los jugadores «bailan» en el pad de baile para introducir comandos. Los pads DDR tienen 4 paneles: Arriba, Abajo, Izquierda y Derecha. Los pads DDR Solo añaden paneles Superior Izquierdo y Superior Derecho. Los pads PIU tienen 5 paneles: Superior Izquierdo, Superior Derecho, Inferior Izquierdo, Inferior Derecho y Centro. También existen pads de 9 paneles con el fin de ser compatibles tanto con PIU como con DDR sin tener que cambiar el pad. Los paneles DDR y PIU normales son usados a menudo en modo doble en sus respectivos juegos, donde un jugador usa 2 pads para una canción. Los juegos DDR Solo eran únicamente de un jugador, por lo que no son dobles.
```

File: wiki/Gameplay/Play_style/es.md [hunk 12]
```diff
@@ -1,141 +1,173 @@
...
 
-### Controlador de DJMAX/O2Jam/Beatmania IIDX/etc
+Una gran desventaja de usar un dance pad es que puede ser difícil presionar varias teclas a la vez. Para pulsar 4 notas en un pad DDR, tendrías que usar tus manos y pies, o posicionar tus pies en 2 paneles a la vez.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:166]:
```suggestion
Una gran desventaja de usar un pad de baile es que puede ser difícil presionar varias teclas a la vez. Para pulsar 4 notas en un pad DDR, tendrías que usar tus manos y pies, o posicionar tus pies en 2 paneles a la vez.
```

File: wiki/Gameplay/Play_style/es.md [hunk 13]
```diff
@@ -1,141 +1,173 @@
...
 
-El estilo de los veteranos en otros juegos de ritmo similares: se usa un controlador arcade en lugar de teclado.
+Otro inconveniente es que los mapas de osu!mania pueden haberse diseñado sin tener en cuenta el movimiento de las animaciones de los juegos de baile, lo que limita la cantidad de canciones que se pueden bailar (cómodamente). Los juegos de baile a menudo están trazados con el movimiento un flujo de animaciones de baile. Una gran regla del trazado en un juego de baile como DDR es ser capaz de alternar los pies en cada nota (con algunas excepciones). Ya que algunos mapas de osu!mania realmente no siguen estas reglas, pueden hacer realmente incómodo jugar muchos mapas en un dance pad.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:168]:
```suggestion
Otro inconveniente es que los mapas de osu!mania pueden haberse diseñado sin tener en cuenta el movimiento de las animaciones de los juegos de baile, lo que limita la cantidad de canciones que se pueden bailar (cómodamente). Los juegos de baile a menudo están diseñados con el movimiento de un flujo de animaciones de baile. Una de las reglas más importantes de un juego de baile como DDR es poder alternar de pie en cada nota (con algunas excepciones). Ya que algunos mapas de osu!mania no siguen realmente esas reglas, puede resultar realmente incómodo jugar muchos mapas en un pad de baile.
```

File: wiki/Gameplay/Play_style/es.md [hunk 14]
```diff
@@ -1,141 +1,173 @@
...
-- **Batería o guitarra**(5/6K): El controlador por defecto de ([Guitar Hero](https://es.wikipedia.org/wiki/Guitar_Hero_(serie))) o ([Drummania](https://es.wikipedia.org/wiki/GITADORA_(serie))). Los jugadores utilizan los botones de la guitarra o la batería. Los jugadores que utilicen batería tendrán la desventaja de que solo pueden utilizar dos notas a la vez sin un pedal especial.
-- **Controlador de Para Para**(5K): El controlador de [Para Para Paradise](https://es.wikipedia.org/wiki/ParaParaParadise_(serie)). Los jugadores mueven sus manos, codos y manos frente a los sensores de proximidad.
+- **Set de guitarra/batería**(5/6K): El controlador por defecto de ([Guitar Hero](https://es.wikipedia.org/wiki/Guitar_Hero_(serie))) o ([Drummania](https://es.wikipedia.org/wiki/GITADORA_(serie))). Los jugadores utilizan los botones de la guitarra o la batería. Los jugadores que utilicen batería tendrán la desventaja de que solo pueden utilizar dos notas a la vez sin un pedal especial.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:172]:
```suggestion
- **Conjunto de guitarra/batería** (5/6K): El mando predeterminado de ([Guitar Hero](https://es.wikipedia.org/wiki/Guitar_Hero_(serie))) o ([Drummania](https://es.wikipedia.org/wiki/GITADORA_(serie))). Los jugadores usan los botones de la guitarra o la batería. Los jugadores que usen la batería tendrán la desventaja de que solo pueden usar dos notas a la vez sin un pedal especial.
```

File: wiki/Gameplay/Play_style/es.md [hunk 15]
```diff
@@ -1,141 +1,173 @@
...
+- **Controlador de Para Para**(5K): Este es el controlador por defecto de la serie de juegos [Para Para Paradise](https://es.wikipedia.org/wiki/ParaParaParadise_(serie)). Los jugadores mueven sus brazos, codos y manos frente a los sensores de proximidad para introducir comandos basados en la posición a través de dichos sensores. La desventaja es la sensibilidad del receptor de entrada y la limitación física del jugador.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:173]:
```suggestion
- **Mando de Para Para** (5K): Este es el mando predeterminado de la serie de juegos [Para Para Paradise](https://es.wikipedia.org/wiki/ParaParaParadise_(serie)). Los jugadores mueven sus brazos, codos y manos frente a los sensores de proximidad para introducir comandos basados en la posición a través de dichos sensores. La desventaja es la sensibilidad del receptor de entrada y la limitación física del jugador.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Comment by agatemosu** [comment]
46

File: wiki/Gameplay/Play_style/es.md [hunk 16]
```diff
@@ -1,141 +1,173 @@
...
 
-### Controlador de DJMAX/O2Jam/Beatmania IIDX/etc
+Una gran desventaja de usar un pad de baile es que puede ser difícil presionar varias teclas a la vez. Para pulsar 4 notas en un pad DDR, tendrías que usar tus manos y pies, o posicionar tus pies en 2 paneles a la vez.
 
-El estilo de los veteranos en otros juegos de ritmo similares: se usa un controlador arcade en lugar de teclado.
+Otro inconveniente es que los mapas de osu!mania pueden haberse diseñado sin tener en cuenta el movimiento de las animaciones de los juegos de baile, lo que limita la cantidad de canciones que se pueden bailar (cómodamente). Los juegos de baile a menudo están diseñados con el movimiento de un flujo de animaciones de baile. Una de las reglas más importantes de un juego de baile como DDR es poder alternar de pie en cada nota (con algunas excepciones). Ya que algunos mapas de osu!mania no siguen realmente esas reglas, puede resultar realmente incómodo jugar muchos mapas en un pad de baile.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:168]:
```suggestion
Otro inconveniente es que los mapas de osu!mania pueden haberse diseñado sin tener en cuenta los movimientos que se realizan en los juegos de baile, lo que limita la cantidad de canciones que se pueden bailar (cómodamente). Los juegos de baile a menudo están diseñados con los movimientos de un flujo de animaciones de baile. Una de las reglas más importantes de un juego de baile como DDR es poder alternar de pie en cada nota (con algunas excepciones). Ya que algunos mapas de osu!mania no siguen realmente esas reglas, puede resultar realmente incómodo jugar muchos mapas en un pad de baile.
```

File: wiki/Gameplay/Play_style/es.md [hunk 17]
```diff
@@ -1,141 +1,173 @@
...
-- **Batería o guitarra**(5/6K): El controlador por defecto de ([Guitar Hero](https://es.wikipedia.org/wiki/Guitar_Hero_(serie))) o ([Drummania](https://es.wikipedia.org/wiki/GITADORA_(serie))). Los jugadores utilizan los botones de la guitarra o la batería. Los jugadores que utilicen batería tendrán la desventaja de que solo pueden utilizar dos notas a la vez sin un pedal especial.
-- **Controlador de Para Para**(5K): El controlador de [Para Para Paradise](https://es.wikipedia.org/wiki/ParaParaParadise_(serie)). Los jugadores mueven sus manos, codos y manos frente a los sensores de proximidad.
+- **Conjunto de guitarra/batería** (5/6K): El mando predeterminado de ([Guitar Hero](https://es.wikipedia.org/wiki/Guitar_Hero_(serie))) o ([Drummania](https://es.wikipedia.org/wiki/GITADORA_(serie))). Los jugadores usan los botones de la guitarra o la batería. Los jugadores que usen la batería tendrán la desventaja de que solo pueden usar dos notas a la vez sin un pedal especial.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:172]:
```suggestion
- **Conjunto de guitarra/batería** (5/6K): El mando predeterminado de ([Guitar Hero](https://es.wikipedia.org/wiki/Guitar_Hero_(serie))) o ([Drummania](https://es.wikipedia.org/wiki/GITADORA_(serie))). Los jugadores tocan la guitarra o la batería y los platillos para introducir una serie de comandos. Los que tocan la batería están en desventaja en comparación con los que tocan la guitarra, porque solo pueden introducir dos comandos a la vez sin un pedal especial.
```

File: wiki/Gameplay/Play_style/es.md [hunk 18]
```diff
@@ -1,141 +1,173 @@
----
-outdated_translation: true
----
+# Estilo de juego
```
Comment (UnknownOsu) [inline wiki/Gameplay/Play_style/es.md:1]:
```suggestion
# Estilos de juego
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**